packages/devtools-launchpad/bin/development-server.js

```diff
@@ -140,6 +143,15 @@ function handleGetConfig(req, res) {
   res.json(getConfig());
 }
 
+function handleSetConfig(req, res) {
+  const name = req.body.name;
+  const value = req.body.value;
+  setValue(name, value);
+  const output  = updateLocalConfig(root);
+  console.log(output);
+  res.end(output);
+}
```

Comment:
within `handleSetConfig` we can implement logic to restart the dev-server programatically.
Here's a completely rough sketch which probably doesn't work.

````javascript
// inside handleSetConfig
app.close('restart');

// outside
app.listen(serverPort, "0.0.0.0", onRequest);
app.on('close', data=> data == "restart" ? app.listen(serverPort, "0.0.0.0", onRequest) : null;
````